src/gulcalc/gulcalc.cpp

```diff
@@ -258,66 +456,6 @@ void gulcalc::outputcoveragedata(int event_id)
 	cov_.resize(coverages_->size());
```

Comment:
If the size is smaller than the current size, should this step be skipped?

Reply:
I'm not sure. This is used for a deprecated output stream. I can take a look but considering this PR forms part of a patch, I would prefer to submit this PR and create another issue regarding the above. Would you agree?

Reply:
If it's deprecated, then I agree. Leave it as is.